Database/PrioriDbContext.cs

```diff
@@ -1,5 +1,6 @@
 using Microsoft.EntityFrameworkCore;
 using PRIORI_SERVICES_API.Model;
```

Comment:
Deveria remover isso aqui, ou só renomear mesmo :)
```suggestion
using PRIORI_SERVICES_API.Models;
```